graphglue-core/src/main/kotlin/io/github/graphglue/model/property/BasePropertyDelegate.kt

```diff
@@ -24,14 +25,17 @@ import kotlin.reflect.KProperty1
  * @param T the type of the value [Node] (s)
  * @param R the type of property
  */
-abstract class BasePropertyDelegate<T : Node?, R>(protected val parent: Node, protected val property: KProperty1<*, *>) {
+abstract class BasePropertyDelegate<T : Node?, R>(
+    protected val parent: Node,
+    protected val property: KProperty1<*, *>
+) {
 
     /**
      * Cache for database results for specific query parts
      */
     private val cache = IdentityHashMap<NodeQueryOptions, NodeQueryResult<T>>()
 
-    private val lazyLoadingDelegate = object: LazyLoadingDelegate<T, R> {
```

Comment:
once again i ask for a pipeline format checker

Reply:
once again, I can tell you that the formatter is not stable, therefore if you find a good solution, you're welcome to open a PR